src/translators/engines/deepl.ts

```diff
@@ -26,12 +26,26 @@ deepl.interceptors.request.use((req) => {
     ? 'https://api-free.deepl.com/v2'
     : 'https://api.deepl.com/v2'
 
-  req.params = {
-    auth_key: Config.deeplApiKey,
+  // Just add to payload... (not a param, then everything needs to be in params)
```

Comment:
This might be a remark showing my ignorance on the subject. In my defence: in the http rest extension I was using in vscode I had to do that - put the api key in de request body, in stead of the making it part of the request url as parameter (even though DeepL's example includes both. That actually didn't work for me).